apps/site/assets/js/app.js

```diff
@@ -46,6 +48,13 @@ import eventPageSetup from "./event-page-setup";
 import previousEventsButton from "./view-previous-events";
 import pslPageSetup from "./psl-page-setup.js";
 
+if (window.sentry) {
+  Sentry.init({
+    dsn: window.sentry.dsn,
+    integrations: [new BrowserTracing()]
+  });
```

Comment:
**suggestion (non-blocking):** You may also want to [pass an environment](https://docs.sentry.io/platforms/javascript/configuration/environments/) option here to more easily differentiate between your prod and your dev errors.

Reply:
An example of passing the environment in Glides: [frontend](https://github.com/mbta/glides/pull/431/files#diff-2692177d3a0965491f6ceb040f13619e4b9200e106bf09f3c2b7d7227f700b8dR28) [backend](https://github.com/mbta/glides/blob/babb0b117f62f6943a1b397421e66c89ba8b5812/config/prod.exs#L97)

Reply:
@skyqrose this example is 💯 
I think I'll set this up in a followup PR.